cameo/api/products.py

```diff
@@ -70,7 +73,15 @@ def search(self, query):
         if len(matches) > 0:
             return matches
         else:
-            raise Exception("No compound matches found for query %s" % query)
```

Comment:
Because not finding something is not an error. I prefer to return an empty result.